firecrest/BasicClient.py

```diff
@@ -625,7 +625,7 @@ def simple_download(self, machine, source_path, target_path):
         with context as f:
             f.write(resp.content)
 
-    def simple_upload(self, machine, source_path, target_path):
+    def simple_upload(self, machine, source_path, target_path, filename):
```

Comment:
```suggestion
    def simple_upload(self, machine, source_path, target_path, filename=None):
```
Let's add a default value of `None` so that it doesn't break old clients and the unittests

Reply:
@ekouts Thanks! Changed.